pkl-cli/pkl-cli.gradle.kts

```diff
@@ -190,13 +201,15 @@ fun Exec.configureExecutable(
         // needed for messagepack-java (see https://github.com/msgpack/msgpack-java/issues/600)
         add("--initialize-at-run-time=org.msgpack.core.buffer.DirectBufferAccess")
         add("--no-fallback")
+        // must be emitted before any experimental options are used
+        add("-H:+UnlockExperimentalVMOptions")
         add("-H:IncludeResources=org/pkl/core/stdlib/.*\\.pkl")
         add("-H:IncludeResources=org/jline/utils/.*")
         add("-H:IncludeResourceBundles=org.pkl.core.errorMessages")
         add("-H:IncludeResources=org/pkl/commons/cli/PklCARoots.pem")
-        add("--macro:truffle")
```

Comment:
(1) renamed version of this is now `--macro:truffle-svm`
(2) it is now applied automatically for builds with Truffle on the modulepath (see [here](https://github.com/oracle/graal/blob/0e2520d037b3a70dd496d3658b877446ba21a182/truffle/src/com.oracle.truffle.api/src/META-INF/native-image/org.graalvm.truffle/truffle-api/native-image.properties#L3))